src/core/token.cpp

```diff
@@ -0,0 +1,17 @@
+#include "token.hpp"
+#include <openssl/rand.h>
+#include <boost/uuid/uuid_generators.hpp>
+#include <boost/uuid/uuid_io.hpp>
+
+std::string Token::createToken(const std::string& username, const std::string& email) {
+    unsigned char randomChars[16];
+    RAND_bytes(randomChars, sizeof(randomChars));
+
+    std::string input(reinterpret_cast<char*>(randomChars), sizeof(randomChars));
```

Comment:
How is this used?

Reply:
to add random characters before converting to uuid to prevent you from generating this token yourself